scripts/requirements-extras.txt

```diff
@@ -17,3 +17,7 @@ Pillow
 
 # can be used to sign a Zephyr application binary for consumption by a bootloader
 imgtool>=1.7.1
+
+# used by nanopb module to generate sources from .proto files
+protobuf
+grpcio-tools
```

Comment:
Unfortunately this turned out to not be sufficient for Windows users, so there is still need to ensure that `protoc` is in path to make it easier for user to understand the error.